vmm/src/memory_manager.rs

```diff
@@ -103,6 +110,34 @@ impl MemoryZone {
     pub fn virtio_mem_zone(&self) -> &Option<VirtioMemZone> {
         &self.virtio_mem_zone
     }
+    pub fn virtio_mem_zone_mut(&mut self) -> &mut Option<VirtioMemZone> {
+        &mut self.virtio_mem_zone
+    }
+    pub fn active_ranges(&self) -> Vec<(u64, u64, u64)> {
```

Comment:
Why is this `active_ranges` ? Can you have inactive ones?

Reply:
This is a function to return the list of ranges that are "plugged". Let's say you have a region that is half plugged, this function will return a range that is half of the full region.
Do you want me to call it `plugged_ranges()` instead, or is `active_ranges()` ok?

Reply:
I think this is fine.